src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -2815,6 +2812,9 @@ private int[] SerializeMethodBodies(BlobBuilder ilBuilder, PdbWriter pdbWriterOp
 
             var encoder = new MethodBodiesEncoder(ilBuilder);
 
+            var mvidStringHandle = default(UserStringHandle);
```

Comment:
After reading the code below, I see that this value is used to indicate whether the MVID string has been allocated already. And it is very closely linked with the mvidStringFixup being assigned. Consider adding a comment. Also, since these two values are closely related consider combining them in a struct (KeyValuePair would work for example).

---
In reply to: [63575659](https://github.com/dotnet/roslyn/pull/11357#discussion_r63575659) [](ancestors = 63575659)